### DISCUSSION TITLE
Significant mismatch between official Swiss vs. electricitymaps electricity consumption data

### DISCUSSION THREAD

**Discussion by delroth** [electricitymaps/electricitymaps-contrib#4569] [Q&A]
Hi folks,

A few weeks ago the Swiss federal council released their official 2021 report for electricity consumption in Switzerland, including breakdown per electricity production type. News release: [FR](https://www.admin.ch/gov/fr/accueil/documentation/communiques.msg-id-90221.html) / [DE](https://www.admin.ch/gov/de/start/dokumentation/medienmitteilungen.msg-id-90221.html). The data significantly differs from [what's currently modeled on ElectricityMaps](https://user-images.githubusercontent.com/202798/190129928-8f83cf1b-c00a-410e-b077-81333daba404.png).

The overall consumption values for 2021 are close enough; ElectricityMaps shows an average of 6.919GW consumed over 2021, which means ~60.6TWh, while the Pronovo report says ~57.1TWh. However the per source breakdown look very different for some sources:

| | ElectricityMaps | Pronovo |
| -- | ---------- | ------- |
| Nuclear | 30.9% | 18.53% |
| Biomass | 1.0% | 0.49% |
| Wind | 0.2% | 2.2% |
| Solar | 5.0% | 1.77% |
| Hydro¹ | 52.4% | 68.44% |
| Coal | 0% | 0.78% |
| Gas | 0% | 0.45% |
| Waste incineration | 0% | 0.62% |
| Misc small renewable electricity producers² | 0% | 6.70% |
| Unknown³ | 10.5% | 0% |

¹ ElectricityMaps breaks down hydro vs. pumped hydro separately, it makes very little sense to me why.
² This is e.g. individual solar panel owners, or small hydro plants below reporting requirements. Estimated: 47.5% hydro, 16.4% solar, 3.1% wind, and 33% biomass or waste from biomass
³ ElectricityMaps estimates this to be 52% hydro storage, 48% unknown thermal with est. 383 gCO2eq/kWh.

Overall this leads Pronovo to conclude that 79.60% of electricity consumed in Switzerland comes from renewable sources, and only 1.87% from fossil fuels. Meanwhile, ElectricityMaps claims that only 52% comes from renewables, with a gCO2eq/kWh of 117g. Very different conclusions.

The electricitymaps website currently says "Exchanges are accounted for in the calculations but not currently shown." but data like 0% coal/gas really makes it look like imports are not taken into account at all.

Any ideas whose data should be "more trusted" here? Are there actual issues with the ElectricityMaps modeling that can be inferred from these very different 2021 averages?

Best,

**Comment by VIKTORVAV99** [comment]
Something to note here:
- Electricity Maps treat biomass and waste as one type (for now at least) so it 1% for Electricity Maps and 1.11% for Pronovo.
- Hydro and Pumped Hydro is separate for a few reasons the mains ones being:
  - It has a vastly different CO2 intensity than damns and rivers as they use grid electricity to pump/store the water.
  - It better shows how pumped hydro is used as a peak shaving/balancing unit over a whole day (charging at night or midday if there is excess solar and discharging at peaks).
- Electricity Maps unknown category contains Misc renewable, wind, gas and coal and adds up to 10.13% for Pronova and 10.5% for Electricity Maps.

So the biggest difference to me seems to be that Electricity Maps is overestimating nuclear and solar and compensate with reducing other controllable production (hydro).

Hopefully the team can use the new official data to improve the models but as far as estimation models go it's pretty good.

Would love to hear your input on the matter @pierresegonne!

**Reply by delroth** [reply]
> Electricity Maps unknown category contains Misc renewable, wind, gas and coal and adds up to 10.13% for Pronova and 10.5% for Electricity Maps.

That seems like the estimation of it being 52% hydro storage and 48% unknown thermal for gCO2eq/kWh calculation is very wrong then, and it should include the right share of small hydro/solar/wind/biomass. This "unknown" category is by far the largest contributor to estimated CO2 emissions in 2021 on electricitymaps, so this seems important to get right.

Assuming that indeed ElectricityMaps "unknown" ends up containing "Misc renewable", "wind", "gas" and "coal" from Pronovo, the 10.5% of ElectricityMaps is constituted of:
- Hydro: 31.4%
- Wind: 23.8%
- Biomass: 21.8%
- Solar: 10.8%
- Coal: 7.7%
- Gas: 4.4%

Pronovo data doesn't break down hydro vs. hydro storage, but for this category it's mostly ROR hydro with very little storage. Let's assume a 80% hydro 20% storage mix, though really I suspect it's less storage than that. With ElectricityMaps gCO2eq/kWh per source numbers, this would translate to "unknown" carbon intensity = 154 gCO2eq/kWh, compared to the current 383 gCO2eq/kWh on ElectricityMaps!

Since "58.91% of emissions from electricity available in Switzerland come from unknown" according to ElectricityMaps, 58.91% of 117 gCO2eq/kWh in 2021 is 68.92 gCO2eq/kWh. If "unknown" is actually 145 gCO2eq/kWh instead, this 68.92 becomes 27.7 gCO2eq/kWh, which means CH 2021 avg is now 76 gCO2eq/kWh instead of 117! Not including the ripple effect of this on how hydro storage carbon intensity is calculated (I guess we'd have to iterate the calculation a few times).

**Reply by VIKTORVAV99** [reply]
Looks like you are correct about the unknown CO2 intensity.

We should try and correct this as soon as possible (assuming all the numbers check out). The good this is that they will be retroactively adjusted if we set the new CO2 intensity as valid from 2021-01-01.

Sadly I don't have time to do this myself right now but if you feel comfortable with accuracy of the production numbers and the calculated CO2 intensity then feel free to do it yourself. I'd be happy to help when I can but I'm quite busy at the moment so might take a while.

**Comment by pierresegonne** [comment]
Hey @delroth,

Thanks a lot for raising this issue. 

* __nuclear__: These values are not modelled. Where do you get 30.9% for Electricity Maps? On the aggregated view, the number is 21.7%. These values come directly from ENTSO-E which in turns obtains them from the Swiss official sources. So total nuclear production over the year should match exactly (the difference in proportion being explained by the difference in total production)
* __unknown make-up__: Indeed, maybe the values we had before were wrongful. All calculations are detailed [here](https://github.com/electricitymaps/electricitymaps-contrib/wiki/CH-Estimation-Model), in which you can get access to the spreadsheet were the calculations were made. Could you please review and propose an update? I would then include it and reprocess the data :)
* __pronovo data__: Looking at the table below 
<img width="981" alt="Screenshot 2022-09-22 at 10 05 29" src="https://user-images.githubusercontent.com/32778266/191692422-997ae8dd-c1b9-4b30-a306-48f52b09dc7e.png">
I'm not sure to understand the distinction between total and Switzerland only. Do you have context on that?

**Comment by sheimers** [comment]
I think Pronovo shows what is sold to customers in Switzerland, not what is produced in Switzerland. So customers get 18.38% of their electricity from swiss nuclear power plants, but the production is higher, used for exports. More detailed statistics for production, import/export and consumtion in Switzerland can be downloaded from Swissgrid.  [https://www.swissgrid.ch/en/home/operation/grid-data/generation.html](https://www.swissgrid.ch/en/home/operation/grid-data/generation.html)

**Reply by sheimers** [reply]
Sorry I was wrong Swissgrid does not tell what kind of power plant is producing the electricity, but you can find it here, nuclear is about 31% of the Swiss electricity production in 2021: [https://www.bfe.admin.ch/bfe/de/home/versorgung/statistik-und-geodaten/energiestatistiken/elektrizitaetsstatistik.html/](https://www.bfe.admin.ch/bfe/de/home/versorgung/statistik-und-geodaten/energiestatistiken/elektrizitaetsstatistik.html/)

**Comment by xiaoshir** [comment]
Hi, as it says in the 

> News release: [FR](https://www.admin.ch/gov/fr/accueil/documentation/communiques.msg-id-90221.html) / [DE](https://www.admin.ch/gov/de/start/dokumentation/medienmitteilungen.msg-id-90221.html)

The 80% of renewable supply is calculated based on the data from the website [stromkennzeichnung](www.stromkennzeichnung.ch), which is based on the purchase of electricity labels (Guarantees of Origins, or so-called GOs) from the market by utility companies. This is totally independent from the actual electricity consumption mix one gets from the socket, and is not appropriate to be used, or at least needs to be carefully considered when performing life cycle assessment of products and systems, as double accounting exists (check the case for Norway and Switzerland here under section [7.2 Consequences of Using the GO Mix in Switzerland](https://treeze.ch/fileadmin/user_upload/downloads/Publications/Case_Studies/Energy/726_LCA_GO-electricityMixes_EuropeanCountries_v1.01.pdf)

So in short, depending on the application you have that requires the electricity supply mix or its life cycle carbon intensity, unless you can confirm to exclude the possibility of double accounting, you should use the data from electricityMap.
